libs/designsystem/src/lib/scss/base/_variables.scss

```
@@ -51,6 +51,7 @@ $font-weight: (
   light: 300,
   normal: 400,
   bold: 700,
+    'black': 900, // black will be treated as a named css color if not quoted
```

Indentation is off...
```suggestion
  'black': 900, // black will be treated as a named css color if not quoted
```